opentok/opentok-meet
# OPENTOK-46538: Remove old anvil end point on meet

#### What is this PR doing?
https://opentok-meet.herokuapp.com/ is making a lot of requests to anvil that are 404ing. See https://vonage.slack.com/archives/CCX1CLQKV/p1645719021635629 & https://vonage.slack.com/archives/CCY5ZBC6T/p1645737904105019 for more context. Apparently this has been happening for a while but was only recently discovered.

Kongpheng and traced this to https://github.com/opentok/opentok-meet/blob/master/plugins/anvil.js which had a subscriber stats report added 6 years ago. As this isn't returning anything useful and is causing problems for the anvil team it is being removed. 

#### How should this be manually tested?
To reproduce on master: 
- Start a local instance of meet running on the prod server. 
- Pub/sub in two different tabs.
- In a new tab make a request to `http://localhost:4000/${roomName}/subscriber/${subscriberId}`
- On kibana prod use the query `call_type:"SessionResource.getSubscriber" AND sessionId:"${sessionId}"` in anvil_access. Verify that you see an event with an httpStatus of 404.

To test on this branch:
- Start a local instance of meet running on the prod server. 
- Pub/sub in two different tabs.
- In a new tab make a request to `http://localhost:4000/${roomName}/subscriber/${subscriberId}`
- On kibana prod use the query `call_type:"SessionResource.getSubscriber" AND sessionId:"${sessionId}"` in anvil_access. Verify that you don't see any events. 


#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-46538
